docs/docs/integrations/providers/databricks.md

```diff
@@ -25,17 +24,60 @@ Databricks provides a fully managed and hosted version of MLflow integrated with
-
 Databricks as an LLM provider
 -----------------------------
 
 The notebook [Wrap Databricks endpoints as LLMs](/docs/integrations/llms/databricks) illustrates the method to wrap Databricks endpoints as LLMs in LangChain. It supports two types of endpoints: the serving endpoint, which is recommended for both production and development, and the cluster driver proxy app, which is recommended for interactive development. 
 
-Databricks endpoints support Dolly, but are also great for hosting models like MPT-7B or any other models from the Hugging Face ecosystem. Databricks endpoints can also be used with proprietary models like OpenAI to provide a governance layer for enterprises.
+Databricks External Models
+--------------------------
+
+[Databricks External Models](https://docs.databricks.com/generative-ai/external-models/index.html) is designed to streamline the usage and management of various large language model (LLM) providers, such as OpenAI and Anthropic, within an organization. It offers a high-level interface that simplifies the interaction with these services by providing a unified endpoint to handle specific LLM related requests. The following example creates an endpoint that serves OpenAI's GPT-4 and generates a chat response from it:
+
+```python
+from langchain.chat_models import ChatDatabricks
+from langchain.schema.messages import HumanMessage
+from mlflow.deployments import get_deploy_client
+
+
+client = get_deploy_client("databricks")
+name = f"chat"
+client.create_endpoint(
+    name=name,
+    config={
+        "served_entities": [
+            {
+                "name": "test",
+                "external_model": {
+                    "name": "gpt-4",
+                    "provider": "openai",
+                    "task": "llm/v1/chat",
+                    "openai_config": {
+                        "openai_api_key": "{{secrets/<scope>/<key>}}",
+                    },
+                },
+            }
+        ],
+    },
+)
+chat = ChatDatabricks(endpoint=name, temperature=0.1)
+print(chat([HumanMessage(content="hello")]))
+# -> content='Hello! How can I assist you today?'
+```
+
+Databricks Foundation Model APIs
+--------------------------------
+
+[Databricks Foundation Model APIs](https://docs.databricks.com/machine-learning/foundation-models/index.html) which allow you to access and query state-of-the-art open source models from a dedicated serving endpoint. With Foundation Model APIs, developers can quickly and easily build applications that leverage a high-quality generative AI model without maintaining their own model deployment. The following example uses the `databricks-bge-large-en` endpoint to embed text:
+
+```python
+from langchain.llms import DatabricksEmbeddings
+
+
+embeddings = DatabricksEmbeddings(endpoint="databricks-bge-large-en")
+print(embeddings.embed_query("hello")[:3])
+# -> [0.051055908203125, 0.007221221923828125, 0.003879547119140625, ...]
+```
 
 Databricks Dolly
```

Comment:
Not sure if this is something we still want to highlight, but no harm to keep it.

Reply:
Let's remove

Reply:
removed